gen_import_utils.py

```diff
@@ -142,3 +142,8 @@ def generate_token(timestamp, filename):
     mac = hmac.new(settings.KEY.encode(), timestamp.encode() + filename.encode(), digestmod='md5')
     print(f"Generated new token for {filename} at {timestamp}.")
     return ':'.join((mac.hexdigest(), timestamp))
+
```

Comment:
used as a more flexible alternative to using row[column_name] for the update class , as it is usable with dataframes that may or may not contain the column. e.g if you just want a csv that will update UTM but not TNRS fields, the updater class won't return a key error.